OpenRA.Game/FileSystem/FileSystem.cs

```diff
@@ -243,31 +233,13 @@ public static bool Exists(string name)
 				return MountedFolders.Any(f => f.Exists(name));
 		}
 
-		static Dictionary<string, Assembly> assemblyCache = new Dictionary<string, Assembly>();
-
 		public static Assembly ResolveAssembly(object sender, ResolveEventArgs e)
 		{
 			foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
 				if (assembly.FullName == e.Name)
 					return assembly;
 
-			var frags = e.Name.Split(',');
```

Comment:
In any case, i'm pretty sure that the check above is effectively a no-op, so if we do remove this code below then we may as well remove the method and the event entirely.